### PR TITLE
Update Expression Statement definition

### PR DESCRIPTION
**Pull request by apastan**
Please, consider the offered definition for JavaScript `Expression Statement`. It looks more clear to me.

I picked up this definition from [here](https://2ality.com/2012/09/expressions-vs-statements.html).

<blockquote>
Wherever JavaScript expects a statement, you can also write an expression. Such a statement is called an expression statement. The reverse does not hold: you cannot write a statement where JavaScript expects an expression.
</blockquote>

### CODE REVIEW

**Comment by Josh-Cena** [comment]
A definition for a statement must follow the structure of "The **XXX** statement [does something]". Could you find a way to rephrase your definition? Honestly I find it to contain information less vital than before.

File: files/en-us/web/javascript/reference/statements/expression_statement/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ spec-urls: https://tc39.es/ecma262/multipage/ecmascript-language-statements-and-
 
 {{jsSidebar("Statements")}}
 
-An **expression statement** evaluates an expression and discards its result. It allows the expression to perform side effects, such as executing a function or updating a variable.
+Wherever JavaScript expects a statement, you can also write an expression. Such a statement is called an **expression statement**. The reverse does not hold: you cannot write a statement where JavaScript expects an expression.
```
Comment (apastan) [inline files/en-us/web/javascript/reference/statements/expression_statement/index.md:10]:
```suggestion
An **expression statement** is an expression that is used in a JavaScript program wherever a statement is expected. An expression statement evaluates an expression and discards its result therefore it makes sense only for expressions which have side effects, such as executing a function or updating a variable.
```

Comment (apastan) [inline files/en-us/web/javascript/reference/statements/expression_statement/index.md:10]:
```suggestion
An **expression statement** is an expression that is used in the code of a JavaScript program where a statement is expected. An expression statement evaluates an expression and discards its result therefore it makes sense only for expressions which have side effects, such as executing a function or updating a variable.
```

**Review by apastan — COMMENTED** [review]
Please take a look at the following definition versions.

File: files/en-us/web/javascript/reference/statements/expression_statement/index.md (hunk 1 above)
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/statements/expression_statement/index.md:10]: Thanks 👍 My suggestion, based on yours:

```suggestion
An **expression statement** is an expression used in a place where a statement is expected. The expression is evaluated and its result is discarded — therefore, it makes sense only for expressions that have side effects, such as executing a function or updating a variable.
```

**Comment by apastan** [comment]
Thanks for help!

**Review by Josh-Cena — APPROVED** [review]
Thanks, this looks great :)

**Pull request merged**